vnext/src/RNTester/js/examples/RTL/RTLExample.windows.js

```diff
@@ -0,0 +1,749 @@
+/**
```

Comment:
Could you mark the Windows specific changes with "// [windows" etc?

Reply:
they already do have // [windows]